styleguide/derek/templates/thought-leadership-overview/article.ejs

```diff
@@ -47,19 +47,15 @@
     </div>
   </div>
 
-  <hr class="rsTl-divider" />
-
+  <hr class="rsTl-divider-top-quote" />
   <div class="rsTl-article-row">
-    <div class="rsTl-column-quote">
+
+    <blockquote class="rsTl-column-quote">
```

Comment:
I know it's contrary to what we normally do but we actually want to style the `<blockquote>` element here without having to provide a class. The reason is we're going to have a lot of people editing content here, who won't know how to apply these classes. 

Also, we shouldn't require users to add `<hr>` tags either.  We can accomplish the same effect by applying a `border-top` and a `border-bottom` to the `<blockquote>` tag. 

The caveat, is we only want this apply to blockquotes for thought leadership nodes.  To do that, we'd probably want to target something like this: 

```
.rsTl-article-row blockquote
```